stellar_rust_sdk/src/assets/all_assets_request.rs

```diff
@@ -51,7 +54,7 @@ pub struct AllAssetsRequest {
 
     /// Specifies the maximum number of records to be returned in a single response.
     ///   The range for this parameter is from 1 to 200. The default value is set to 10.
-    limit: Option<u32>,
+    limit: Option<u8>,
```

Comment:
Onderzoeksvraagje: kunnen we ook members aan een struct toevoegen middels de DeriveMacro?